packages/rpc-methods/src/restricted/manageAccounts.ts

```diff
@@ -0,0 +1,128 @@
+import {
+  SubjectType,
+  PermissionType,
+  RestrictedMethodOptions,
+  ValidPermissionSpecification,
+  PermissionSpecificationBuilder,
+} from '@metamask/permission-controller';
+import { Json, JsonStruct, NonEmptyArray } from '@metamask/utils';
+import {
+  assert,
+  string,
+  object,
+  optional,
+  union,
+  array,
+  record,
+} from 'superstruct';
+
+type Message = { method: string; params: Json };
```

Comment:
```suggestion
type Message = Infer<typeof SnapMessageStruct>;
```
Can we do this instead?

Reply:
Yes, added in [d1e0834](https://github.com/MetaMask/snaps/pull/1290/commits/d1e0834275d528b96d034aea1b4d2eae620ef847)